docs/inference_helpers/inference_sdk.md

```diff
@@ -585,13 +569,17 @@ The following fields are passed to API
   `disable_preproc_static_crop` to alter server-side pre-processing
 - `disable_active_learning` to prevent Active Learning feature from registering the datapoint (can be useful for
   instance while testing model)
+- `active_learning_target_dataset` - making inference from specific model (let's say `project_a/1`), when we want
+to save data in another project `project_b` - the latter should be pointed by this parameter. **Please remember that
+you cannot use different type of models in `project_a` and `project_b` - if that is the case - data will not be 
+registered) - since `v0.9.18`
 
-* `visualize_predictions`: flag to enable / disable visualisation
-* `confidence_threshold` as `confidence`
-* `stroke_width`: width of stroke in visualisation
-* `disable_preproc_auto_orientation`, `disable_preproc_contrast`, `disable_preproc_grayscale`,
+- `visualize_predictions`: flag to enable / disable visualisation
+- `confidence_threshold` as `confidence`
+- `stroke_width`: width of stroke in visualisation
+- `disable_preproc_auto_orientation`, `disable_preproc_contrast`, `disable_preproc_grayscale`,
   `disable_preproc_static_crop` to alter server-side pre-processing
-* `disable_active_learning` to prevent Active Learning feature from registering the datapoint (can be useful for instance while testing model)
+- `disable_active_learning` to prevent Active Learning feature from registering the datapoint (can be useful for instance while testing model)
```

Comment:
```suggestion
- `disable_active_learning` to prevent Active Learning feature from registering the datapoint (can be useful, for instance, while testing the model)
```